packages/jest-environment-jsdom/src/index.js

```diff
@@ -32,6 +33,16 @@ class JSDOMEnvironment {
     this.global.Error.stackTraceLimit = 100;
     installCommonGlobals(global, config.globals);
 
+    if (!global.requestAnimationFrame) {
+      global.requestAnimationFrame = callback => {
+        const hr = process.hrtime(jsdomInitialized);
+        const hrInNano = hr[0] * 1e9 + hr[1];
+        const hrInMicro = hrInNano / 1e6;
+
+        return global.setTimeout(callback, 0, hrInMicro);
```

Comment:
I'm currently working on getting it upstreamed into JSDOM, so I read some of the spec. I don't know if people use this argument, or expect it to be in any way accurate, but this is a best effort 🙂